app/src/androidTest/java/com/github/se/orator/ui/friends/FriendsUITests.kt

```diff
@@ -246,11 +264,6 @@ class FriendsUITests {
     verify(mockNavigationActions).navigateTo(eq(Screen.LEADERBOARD))
   }
 
-  /**
-   * Tests that the search functionality works correctly on the View Friends screen. It verifies
-   * that the search field exists and is displayed, can be clicked and text can be inputted. After
-   * inputting text, it checks that the first friend item is displayed and the second is not.
-   */
```

Comment:
Hi, the tests you added are great, thanks for your work! I was just wondering if you could revert the changes made to delete comments of the already present tests. I think this would enhance code readability.